Handle `UncatchableError` thrown by current step.
